src/templates/view.html

```
@@ -38,8 +38,14 @@ <h1>
 {% include('view-edit-toolbar.html') %}
 
 <div id='isTimestampedByInfoDiv'>
-{% if Entity.entityData.timestamped %}
-  {{ 'Entry was timestamped by %s on %s at %s'|trans|format(timestamperFullname, Entity.entityData.timestamped_at|date('Y-m-d'), Entity.entityData.timestamped_at|date('H:i:s'))|msg('ok', false) }}
+  {% if Entity.entityData.timestamped %}
+    {{ 'Entry was timestamped by %s on %s at %s'|trans|format(timestamperFullname, Entity.entityData.timestamped_at|date('Y-m-d'), Entity.entityData.timestamped_at|date('H:i:s'))|msg('ok', false) }}
+  {% endif %}
+</div>
+
+<div id='isSignedByInfoDiv'>
+  {% if Entity.entityData.signature_count > 0%}
```

HTMLHint (1.9.2)

[error] Special characters must be escaped : [ > ].

(spec-char-escape)